std_msgs/Header header
DetectedSign[] signs

